src/async_fuse/memfs/kv_engine/mod.rs

```diff
@@ -135,6 +136,8 @@ pub enum KeyType {
     /// Node list
     /// The corresponding value type is ValueType::RawData
     FileNodeList(INum),
+    /// Just a string key for testing the KVEngine.
+    String(String),
```

Comment:
If it's only for test, shall we add `#[test]`?

Reply:
Yes, I am sorry for my cursoriness.